platform/viewer/public/config/public_dicomweb.js

```diff
@@ -1,5 +1,6 @@
 window.config = {
   routerBasename: '/',
+  whiteLabelling: {},
```

Comment:
Is this required, or are we adding it just so users know it's an option?

Reply:
Before it was working without it because we had a default prop for whiteLabelling, now we don't because of the config which its a function return now.

Reply:
After the function return, we can still `Object.assign({}, outputConfig, defaultConfig)` if we need to support defaults.

Reply:
Alright, done. Thanks.